packages/dependency-royalty-management/contracts/RoyaltyDistributer.sol

```diff
@@ -14,16 +14,13 @@ contract RoyaltyDistributer is IERC2981Upgradeable {
 
```

Comment:
This file should probably be named 

`RoyaltyDistributor` instead of
`RoyaltyDistributer`

Reply:
Same for the `MultiRoyaltyDistributer.sol`